.github/workflows/sims.yml

```diff
@@ -16,18 +16,17 @@ on:
     types: [created, edited]
 
 jobs:
-  build:
+  member:
```

Comment:
add `member` job so that membership authentication could be used by other jobs.
this follows the chain-main approach: https://github.com/crypto-org-chain/chain-main/blob/7e7d49839283f6d77516f614c2e2ff7441771e5a/.github/workflows/build.yml#L13